README.md

```diff
@@ -1,6 +1,6 @@
 # [AppMetrica SDK](https://appmetrica.io)
 
-AppMetrica is a free real-time ad tracking and mobile app analytics solution. AppMetrica covers the three key features for discovering your app's performance — ad tracking, usage analytics and crash analytics.
+AAppMetrica is a one-stop marketing platform for install attribution, app analytics, and push campaigns. AppMetrica covers the three key features for discovering your app's performance — ad tracking, usage analytics and crash analytics.
```

Comment:
Hi! Thank you for your commitment! But you have a typo in your edits:
"AAppMetrica".